packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -1,25 +1,48 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
 import '../src/index.ts';
+import { getDisplayCode } from '../../../../../.storybook/common';
 
-const autocompleteDataReceivedEvent = new CustomEvent('autocomplete_received_results',
+// @TODO allow for sending event with searchbar ID. This should allow for one
+// story's events to not affect another story.
+const autocompleteDataReceivedEvent = new CustomEvent('sfx::autocomplete_received_results',
   { detail: [
       { "title": "Brands", "items": [{ "label": "Cats" }, { "label": "Dogs" }] },
       { "title": "default", "items": [{ "label": "Cars" }, { "label": "Bikes" }] }
     ],
     bubbles: true }
 );
 
+function getSayt(searchbar='', showSayt=true): string {
+  const showAttribute = boolean('visible', showSayt) ? 'visible' : '';
+  const closeText = text('Close link text', 'Close');
+  const showCloseButton = boolean('Show Close button', true) ? 'showclosebutton' : '';
+
+  return `<sfx-sayt${searchbar ? ` searchbar="${searchbar}"` : ''}
```

Comment:
spaces after { and before }

Reply:
also maybespace after the tag opening and or bring the first ${searchbar onto the next line? looks cramped

Reply:
This looks very cramped because I need to keep whitespace to a minimum. It will be going into a `<pre>` tag, where whitespace is respected, so any extra newlines inside the string cause newlines to appear on the page.

Reply:
For interpolated strings, we generally haven't been putting padding inside the interpolated portion.